inst/support/report_dataset.Rmd

```diff
@@ -614,8 +614,8 @@ writeLines(
 for(trait in traits_categorical) {
  
   transforms <- metadata$traits %>% util_list_to_df2() %>% filter(trait_name == trait)
-  
-  if(is.na(metadata$substitutions)) {
+
+  if(is.na(metadata$substitutions[1])) {
```

Comment:
What does this change do?

Reply:
This is part of the ongoing small fixes to the reports - `is.na` wants to look at a single cell, but is there are many substitutions then it gets confused. But if there are no substitutions the first cell will be `na` and if there are 1 or more substitutions it won't be.